app/components/createContainer/createcontainer.html

```diff
@@ -267,13 +267,26 @@
               <div class="form-group">
                 <label for="container_network" class="col-sm-1 control-label text-left">Network</label>
                 <div class="col-sm-9">
-                  <select class="selectpicker form-control" ng-model="config.HostConfig.NetworkMode">
```

Comment:
See my comment on why you should use `config.HostConfig.NetworkMode` instead of `network.Mode` here.